pkg/machines/components/storagePools/createStoragePoolDialog.css

```diff
@@ -0,0 +1,3 @@
+.modal-dialog {
+    width: 500px;
```

Comment:
Oh, good point, @andreasn are you ok with this change?

Reply:
Everywhere else I've seen, the dialogs are defined in pixels, but I'm not 100% sure.
@garrett do you know what's best practice here?

Reply:
FTR, this wasn't a veto. I know relatively little about our best practices here, and if we do the same in other places, it's fine.

Reply:
In this case, it would be better to size with `rem`, which is the root em (that is: the default font size, instead of whatever font size might be on the element). However, PatternFly uses `px` sizes for nearly everything, so it's not entirely wrong, especially as a web `px` is not necessarily a device pixel (in other words; a `px` _may_ scale between devices, browsers, or browser overrides). Using `em`/`rem` does size based on font size, however, so you do have predictable amounts of text.

For modals in particular, PatternFly does inherit from Bootstrap, which does have `modal-lg` and `modal-sm` add-on classes for `modal-dialog`, but `modal-sm` is set for a width of `300px`, whereas standard modals are set to `600px`.

_**TL;DR**: Using `rem` might be better but, sure, this works._ :+1:

Reply:
HOWEVER: You _might_ want to not change `.modal-dialog` to 500px, but key it off of something like `.ct-modal-storage` and set that to have width 500px and include that class in addition to `modal-dialog` however. (That is, `className="modal-dialog ct-modal-storage"`)